### PR TITLE
Enable bf16 quantization of models, and fix resetting args.audio in the microservice between runs

### PR DESCRIPTION
**Pull request by ctao456**
## Description

Enable bf16 quantization of models, and fix resetting args.audio in the microservice between runs

## Issues

https://github.com/opea-project/GenAIComps/pull/775

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

https://docs.habana.ai/en/latest/PyTorch/PyTorch_Mixed_Precision/index.html

## Tests

Tests on Xeon, Gaudi

### CODE REVIEW

**Comment by ctao456** [comment]
@chensuyue @lvliang-intel Cleared all checks. Please review. Thanks.

**Review by lvliang-intel — APPROVED** [review]

File: comps/animation/wav2lip/dependency/check_wav2lip_server.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 outfile = os.environ.get("OUTFILE")
 
 # Read the JSON file
-with open("comps/animation/wav2lip/assets/audio/sample_question.json", "r") as file:
```
Comment (Spycsh) [inline comps/animation/wav2lip/dependency/check_wav2lip_server.py:13]: Does the original https://github.com/ctao456/GenAIComps/blob/ctao/opea/comps/animation/wav2lip/assets/audio/sample_question.json need to be removed?

File: comps/animation/wav2lip/dependency/check_wav2lip_server.py [hunk 2]
```diff
@@ -10,9 +10,9 @@
...
+with open("comps/animation/wav2lip/assets/audio/sample_whoareyou.json", "r") as file:
     data = json.load(file)
 
-inputs = {"audio": data["byte_str"]}
+inputs = {"audio": data["byte_str"], "max_tokens": 64}
```
Comment (Spycsh) [inline comps/animation/wav2lip/dependency/check_wav2lip_server.py:16]: Where is "max_tokens" used?

File: comps/animation/wav2lip/dependency/check_wav2lip_server.py (hunk 1 above)
Comment (ctao456) [reply comps/animation/wav2lip/dependency/check_wav2lip_server.py:13]: No. It serves as a sample question to be fed in a post request to the fast API

File: comps/animation/wav2lip/dependency/check_wav2lip_server.py (hunk 2 above)
Comment (ctao456) [reply comps/animation/wav2lip/dependency/check_wav2lip_server.py:16]: It's used in here in the gateway: 
https://github.com/opea-project/GenAIComps/blob/9fec2262613e6db71d617cacf321f84d6971dcc6/comps/cores/mega/gateway.py#L944C9-L952C10

File: comps/animation/wav2lip/dependency/check_wav2lip_server.py (hunk 1 above)
Comment (ctao456) [reply comps/animation/wav2lip/dependency/check_wav2lip_server.py:13]: And it's my personal voice.

**Review by Spycsh — APPROVED** [review]

**Pull request merged**